src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,24 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int MIN_AGE_OF_CANDIDATE = 35;
+    private static final int TOTAL_MIN_PERIOD_LIVE_IN_UKRAINE = 10;
+
+    @Override
+    public boolean test(Candidate candidate) {
+
+        return candidate.getAge() >= MIN_AGE_OF_CANDIDATE
```

Comment:
```suggestion
    public boolean test(Candidate candidate) {
        return candidate.getAge() >= MIN_AGE_OF_CANDIDATE
```


redundant empty line

Reply:
Removed